config/crd/bases/kfdef.apps.kubeflow.org_kfdefs.yaml

```diff
@@ -0,0 +1,187 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
This crd should not be added default `crd` folder.

You can create custom folder. Eg. `additional/kfdef_crd.yaml`

Reply:
done, as `config/crd/additionals/kfdef.apps.kubeflow.org_kfdefs.yaml`